### ISSUE TITLE
Pinch-zoom

### ISSUE THREAD

**Issue by GR1992** [microsoft/vscode#257126]
ADD ISSUE DESCRIPTION HERE

Version: 1.102.1
Commit: 7adae6a56e34cb64d08899664b814cf620465925
User Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0
Embedder: codespaces

I hve an eyesight problem and I rely on browsers built-in pinch-zoom feature a lot. For some reason, this feature does not work in any github coding environments except when viewing .md files, where it works just fine and also zooms in other parts of the page, even though I can not zoom on them itself. Is there any way to override this and pinch-zoom on githubs web IDE and/or workspace?

**Comment by GR1992** [comment]
And the c;ear annoying thing is that this feature is clearly "already there", but disabled. What I mean is, if I ctrl+mousewheelscroll, then the screen zooms exactly as I would want it to - the image gets "stretched" equally in all directions, and I can scroll using sidebar scrolls. 

I just couldn't find how to enable the same behavior using pinch-zoom and touch scroll. It seems like this should be some simple setting switch, and I have seen lots of people asking this before as well, but couldn't find any solutions to it.

**Comment by bpasero** [comment]
cc @meganrogge
